benchmarks/infill_density/data/box_2d_top.0.txt

```diff
@@ -0,0 +1,1002 @@
+# POINTS: 1001
```

Comment:
They are different! The load linearly increases with time, I found this helps stabilize the elastic response instead of dumping the entire load on all at once.